apps/foundation-demo/src/app/helper/routing-guards/cart-load.guard.spec.ts

```diff
@@ -35,43 +36,48 @@ xdescribe('CartLoadGuard', () => {
     cartFactory = TestBed.get(DaffCartFactory);
     stubCart = cartFactory.create();
     store = TestBed.get(Store);
+    spyOn(store, 'dispatch');
   }));
 
   it('should be created', () => {
     expect(cartLoadGuard).toBeTruthy();
   });
 
-  describe('when the cart is null', () => {
```

Comment:
Why did the tests have to change?

Reply:
I didn't think `CartLoad` was _actually_ a guard, so I removed the `CanActivate` function. It doesn't decide if the route Activatable. It just gets the Cart. So `CartLoadGuard` is really more of a helper function file.

Reply:
This isn't relevant anymore. closing